src/util.js

```diff
@@ -66,10 +66,16 @@ const deserialize = (binaryBlob, callback) => {
  * Get the CID of the DAG-Node.
  *
  * @param {ZcashBlock} dagNode - Internal representation of a Zcash block
+ * @param {Object} [options] - Ignored
  * @param {CidCallback} callback - Callback that handles the return value
  * @returns {void}
  */
-const cid = (dagNode, callback) => {
+const cid = (dagNode, options, callback) => {
+  if (options instanceof Function) {
```

Comment:
Searching through the existing IPFS/IPLD code base it seems that the preferred way for checking if something is a function is: ` if (typeof options !== 'function') {`. My guess is that it is related to code minifiers. Please change it.